gef.py

```diff
@@ -3782,7 +3786,7 @@ def reset_architecture(arch: Optional[str] = None, default: Optional[str] = None
         return
 
     try:
-        gef.arch = arches[arch_name]()
+        gef.arch = arches[arch_name.upper()]()
```

Comment:
This is incorrect, some architectures are not uppercase (e.g. `PowerPC`).

Reply:
You are correct. I see the keys in `__registered_architectures__` are not all consistently upper case. I took inspiration for this change [from here](https://github.com/abgeana/gef/blob/5f14edf350c185b70eae240429ae1b323a062a3a/gef.py#L3793), but I see this code is volatile and has already been changed.